app/assets/javascripts/govuk_publishing_components/components/print-link.js

```diff
@@ -0,0 +1,14 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  'use strict'
+
+  Modules.PrintLink = function () {
+    this.start = function (element) {
+      element[0].addEventListener('click', function () {
```

Comment:
Doesn't the following work?

```suggestion
      element.addEventListener('click', function () {
```

Reply:
Ah no :( 
<img width="895" alt="image" src="https://user-images.githubusercontent.com/11051676/85691327-4f67ae80-b6cc-11ea-8081-ac04bd730713.png">

I was a bit confused by this - but appears the the code initialising the modules passes in an array of elements. I've updated the variable name to be plural to reflect this.

Reply:
I'll try and cast a shade of light on this. The module system relies on jQuery (that's how it was implemented and [we're in the process of changing this, but it will require a breaking change](https://github.com/alphagov/govuk_publishing_components/pull/1538)). So the function expects `element` to be a jQuery object, but within the module you wrote you're using plain/vanilla JavaScript and this is the reason you need to use `element[0]`, to get the raw `nodeElement` from a jQuery-wrapped object. So the variable name was correct as it represents a single `element`.

Reply:
Ah thanks @alex-ju, that make a tonne of sense! I've changed the naming back.